changelogs/fragments/update_dependencies.yml

```diff
@@ -0,0 +1,2 @@
+breaking_changes:
```

Comment:
This one also should not be a breaking change. There should be no change from the end user's standpoint.

Reply:
I thought users were constrained to update their dependency versions.

Reply:
Users don't have to update any dependencies. We're distributing a binary that has all the dependencies built into it.  As far as the user is concerned, it's just a new version of the provider that works just like the old version.